docs-new/docs/graphql/core/migrations/config-v2/migrations-setup.mdx

```diff
@@ -193,18 +176,11 @@ git add .
 git commit -m "initialize migrations and metadata"
 ```
 
-<div class="note">
-
-<div class="title">
-
-Note
+:::info Note
 
-</div>
+The version control set up should typically be done right after [Step 2](#step-2-set-up-a-project-directory)
```

Comment:
`migrations-project-init-v2` seems to be correct id from rst file. So, please use same ids and avoid the autogenerated ones from docusaurus.

Also, pls attach these ids to respective headers